Android-Sample/app/src/main/java/com/opentok/qualitystats/sample/MainActivity.java

```diff
@@ -2,407 +2,205 @@
+                new NetworkQualityTestCallbackListener() {
+                    @Override
+                    public void onQualityTestResults(String recommendedSetting) {
+                        Log.d(LOGTAG, "Recommended resolution: " + recommendedSetting);
+                        showRecommendedSettingPopup(recommendedSetting);
+                    }
+
+                    @Override
+                    public void onQualityTestStatsUpdate(CallbackQualityStats stats) {
+                        if (stats == null)
+                            return;
+
+                        String sentVideoInfo = "Sent Video Bitrate: " + stats.getSentVideoBitrateKbps() + " Kbps\n"
+                                + "Sent Audio Bitrate: " + stats.getSentAudioBitrateKbps() + " Kbps \n"
+                                + "Sent Video Resolution: " + stats.getSentVideoResolution() + " \n"
+                                + "Received Video Bitrate: " + stats.getReceivedVideoBitrateKbps() + " Kbps\n"
+                                + "Received Audio Bitrate: " + stats.getReceivedAudioBitrateKbps() + " Kbps\n"
+                                + "Received Video Resolution: " + stats.getReceivedVideoResolution() + "\n"
+                                + "Quality Limitation Reason: " + stats.getQualityLimitationReason() + "\n"
+                                + "Audio Packet Lost Ratio  " + stats.getAudioPacketLostRatio() * 100 + "% \n"
+                                + "Video Packet Lost Ratio  " + stats.getVideoPacketLostRatio() * 100 + "%\n"
+                                + "Current Round Trip Time: " + stats.getCurrentRoundTripTimeMs() + " ms";
+
+                        statsTextView.setText(sentVideoInfo);
+
+                        Log.d(LOGTAG, "---------------------------------------------------------------");
+                        Log.d(LOGTAG, "Sent Video Bitrate: " + stats.getSentVideoBitrateKbps() + " Kbps");
+                        Log.d(LOGTAG, "Sent Audio Bitrate: " + stats.getSentAudioBitrateKbps() + " Kbps");
+                        Log.d(LOGTAG, "Received Audio Bitrate: " + stats.getReceivedAudioBitrateKbps() + " Kbps");
+                        Log.d(LOGTAG, "Received Video Bitrate: " + stats.getReceivedVideoBitrateKbps() + " Kbps");
+                        Log.d(LOGTAG, "Current Round Trip Time: " + stats.getCurrentRoundTripTimeMs() + " ms");
+                        Log.d(LOGTAG, "Available Outgoing Bitrate: " + stats.getAvailableOutgoingBitrate() + " bps");
+                        Log.d(LOGTAG, "Audio Packet Lost Ratio  " + stats.getAudioPacketLostRatio() * 100 + "%");
+                        Log.d(LOGTAG, "Video Packet Lost Ratio  " + stats.getVideoPacketLostRatio() * 100 + "%");
+                        Log.d(LOGTAG, "Jitter: " + stats.getJitter());
+                        Log.d(LOGTAG, "Quality Limitation Reason: " + stats.getQualityLimitationReason());
```

Comment:
Maybe we can change the previous logs with `Log.d(LOGTAG, sentVideoInfo);`. In the future if we have to change something, we only have to change it in one place